### PR TITLE
Outlier detection pipeline

### LINKED ISSUE #2142: Pipelines for outlier detection
A primitive for outlier detection is being added under uncharted-distil/distil-primitives#230.  Pipelines should be created similar to clustering for remote sensing that use the this new primitive to perform outlier analysis.  A pipeline for outlier detection on tabular data should be also be added.

### PR DESCRIPTION
**Pull request by venkatkorapaty**
fixes #2142

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,19 +23,19 @@ postgres:
 #     - $D3MINPUTDIR:$D3MINPUTDIR
 #     - $D3MOUTPUTDIR:$D3MOUTPUTDIR
 
-distil-auto-ml:
```
Comment (cdbethune) [inline docker-compose.yml:26]: probably checked in by mistake?

File: api/routes/outlier_detecting.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			return
+		}
+
+		datasetMeta, err := metaStorage.FetchDataset(dataset, false, false, false)
+		if err != nil {
+			handleError(w, err)
+			return
+		}
+
+		outlierData, err := task.OutlierDetection(datasetMeta, variable)
```
Comment (cdbethune) [inline api/routes/outlier_detecting.go:51]: Skipped the err check

**Review by cdbethune — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (venkatkorapaty) [reply docker-compose.yml]: yes, reverted it.

File: api/routes/outlier_detecting.go (hunk 2 above)
Comment (venkatkorapaty) [reply api/routes/outlier_detecting.go:51]: included it now.

**Review by cdbethune — APPROVED** [review]

**Pull request merged**